### ISSUE TITLE
While using tab key, Table header row is not accessible with keyboard: A11y_Azure Databases extension for VS Code_Keyboard

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/vscode-cosmosdb#2926]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopApp; #BM_Azure Databases extension for VS Code_Win32_Nov2025; #Win32; #Win11; #FTP; #MAS2.1.1; #Keyboard; #Azure Databases extension for VS Code;#Benchmark; #AILimited; #A11yMAS;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**

"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application Name: Azure Databases extension for VS Code
OS Version: Windows 11 enterprise 24H2. (OS build 26120.3360)

### Steps to Reproduce

1. Open VS Code
2. Open the "Azure" Sidebar
3. Sign In to your Azure Account (skip if already signed in)
4. Expand the "Cosmosdb-Portalteam-generaltest-msft" subscription node
5. Expand the "Azure Cosmos DB" node
6. Expand the "a11y-nosql" Cosmos DB account 
7. Expand the "SampleDB" database node
8. Expand the "SampleContainer" node.
9. Tab till "Create query" and press enter.
10. Run a simple query such as "SELECT TOP 1000 * FROM c"
11. Tab till "Results" tab and press enter.
12. Tab till "Table headers".
13. Observed the issue while using tab key, Table header row is not accessible with keyboard.

### Actual Results:
While using tab key, Table header row is not accessible with keyboard.
### Expected Results:
While using tab key, Table header row should be accessible with keyboard.
### User Impact:
Keyboard user will face the issue. They are not able use Resize by content and resize options and if that buttons are not accessible keyboard users will not be able to adjust the table columns.
### Attachment:

https://github.com/user-attachments/assets/f1de9c23-24d0-4e2e-9014-d6ff3ffb4fe0
